src/data-and-backend/google-apis.md

```diff
@@ -5,34 +5,43 @@ description: How to use Google APIs with Flutter.
 
 <?code-excerpt path-base="googleapis/"?>
 
-The [Google APIs package]({{site.pub-pkg}}/googleapis) exposes dozens of Google
+The [Google APIs package][] exposes dozens of Google
 services that you can use from Dart projects.
 
-This page describes how to use APIs that interact with end-user data by using
-Google authentication.
+This page describes how to use APIs that interact with
+end-user data by using Google authentication.
 
 Examples of user-data APIs include
-[Calendar]({{site.pub-api}}/googleapis/latest/calendar_v3/calendar_v3-library.html),
-[Gmail]({{site.pub-api}}/googleapis/latest/gmail_v1/gmail_v1-library.html), and
-[YouTube]({{site.pub-api}}/googleapis/latest/youtube_v3/youtube_v3-library.html).
+[Calendar][], [Gmail][], [YouTube][], and Firebase.
 
 {{site.alert.info}} 
-  Note: The only APIs you should use directly from your Flutter
-  project are those that access user data via Google authentication.
-
-  APIs that require
-  [service accounts](https://cloud.google.com/iam/docs/service-accounts) **should
-  not** be used directly from a Flutter application. Doing so requires shipping
-  service credentials as part of your application, which is not secure. To use
-  these APIs, we recommend creating an intermediate service.
-
-<!-- TODO(kevmoo): Add link to public user guide when available. -->
-
+  The only APIs you should use directly from your Flutter
+  project are those that access user data using Google authentication.
+
+  APIs that require [service accounts][] **should not**
+  be used directly from a Flutter application.
+  Doing so requires shipping service credentials as part
+  of your application, which is not secure.
+  To use these APIs,
+  we recommend creating an intermediate service.
```

Comment:
Do we have an example of how to do this?

Reply:
Sounds like a good cookbook article idea

Reply:
idk

Reply:
I'm also not aware of a sample we have, but a simple sample or cookbook recipe sounds great (https://github.com/flutter/website/issues/9755).

[Cocoon](https://github.com/flutter/cocoon) might use service accounts on its app engine backend, but I am not familiar with it and it's likely too complicated to be a useful example.